piper_bringup/package.xml

```diff
@@ -9,12 +9,13 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <exec_depend>rviz2</exec_depend>
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>gz_ros2_control</exec_depend>
-  <exec_depend>parallel_gripper_controller</exec_depend>
   <exec_depend>joint_state_broadcaster</exec_depend>
   <exec_depend>joint_trajectory_controller</exec_depend>
+  <exec_depend>parallel_gripper_controller</exec_depend>
+  <exec_depend>rviz2</exec_depend>
+  <exec_depend>topic_based_ros2_control</exec_depend>
 
```

Comment:
This also now depends on `piper_single_ctrl`. Can we copy the package over to this repo?